projects/Vula/test.nix

```
@@ -1,21 +1,81 @@
 {
   sources,
-  pkgs,
+  lib,
   ...
-}: {
+}: let
+  inherit (lib) recursiveUpdate;
+in {
   name = "vula";
 
-  nodes = {
-    server = {config, ...}: {
+  nodes = let
+    common = {
       imports = [
         sources.modules.default
         sources.modules."services.vula"
       ];
+
       services.vula.enable = true;
+      services.vula.openFirewall = true;
+
+      services.vula.logLevel = "DEBUG";
     };
+  in {
+    a = recursiveUpdate common {
+      services.vula.userPrefix = "fula";
```

I suggest something that is more obviously a test value, since "fula" is confusing, as evidenced by discussions in this very PR.

```suggestion
      services.vula.userPrefix = "vula-mytestprefix";
```

---

I agree. I was confused and others may also be confused. For example a name that is self-documenting might be a good choice.

---

Amended.

---

@lorenzleutgeb I don't like resolving others' concerns.